src/changelog/unreleased.md

```diff
@@ -12,6 +12,7 @@ on how to add them:
 ### Added
 
 - Add `Window::turbo()`, implemented on X11, Wayland, and Web.
+- Add traits `EventLoopExtWayland` and `EventLoopExtX11`, providing methods `is_wayland` and `is_x11` on `EventLoop`.
```

Comment:
I don't think this goes here, that's the example documentation

Reply:
Good catch, fixed!